posthog/async_migrations/setup.py

```diff
@@ -91,8 +91,8 @@ def kickstart_migration_if_possible(migration_name: str, applied_migrations: set
 
     from posthog.async_migrations.runner import run_next_migration
 
-    # start running 30 minutes from now
-    run_next_migration(migration_name)
+    # start running new migrations 30 minutes from now or continue existing ones immediately
```

Comment:
I'm not sure we have a 30min delay actually, what does https://github.com/PostHog/posthog/blob/master/posthog/async_migrations/utils.py#L58 that delay do?

Reply:
I don't see it actually delaying for 30 minutes? `delay` is just a way to [trigger a celery task](https://docs.celeryproject.org/en/stable/userguide/calling.html#example)

Am I missing something about the 30 minutes delay?

Reply:
I think it was there initially and then got removed in a later PR. We should potentially re-visit the idea anyway. So the comment is wrong.